utils/resource/usage.go

```diff
@@ -84,6 +88,11 @@ func NewManager(diskPath string, frequency, cpuHalflife, diskHalflife time.Durat
 		onClose:            make(chan struct{}),
 		availableDiskBytes: math.MaxUint64,
 	}
+
+	metricsRegisterer := prometheus.NewRegistry()
+	processMetrics, _ := newMetrics("System resources", metricsRegisterer, m.processes)
```

Comment:
the metrics aren't going to be reported correctly by the node if we just use a new registry here - we should be passing in the registry from the node as a param to the `NewManager` call.

Reply:
updated